features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/Configuration.jsx

```diff
@@ -174,6 +176,9 @@ export default function Configuration() {
             case 'tags':
                 nextState[action] = value;
                 return nextState;
+            case 'maxTps':
```

Comment:
I think we could merge this case with the above one since it's the same logic in both cases